db/fixtures/miq_product_features.yml

```diff
@@ -5108,6 +5108,92 @@
           :description: Edit Job Templates Tags
           :feature_type: control
           :identifier: configuration_script_tag
+# Ansible Tower
+- :name: Ansible Tower
+  :description: Everything under Ansible Tower
+  :feature_type: node
+  :identifier: ansible_tower
```

Comment:
@fryguy - I agree - I had Automation Manager first, then discussed with @dclarizio and changed it to ansible_tower. I can change this to automation manager rather than external, and we can name the internal something else.